server/aws-lsp-codewhisperer/src/language-server/agenticChat/qAgenticChatServer.ts

```diff
@@ -47,7 +47,7 @@ export const QAgenticChatServer =
                             ],
                         },
                         mcpServers: enabledMCP(params),
-                        modelSelection: true,
+                        modelSelection: enabledModelSelection(params),
```

Comment:
We need to leave this set to true so that the current extensions that support model selection, but do not emit the model selection capability don't lose their model selection feature.

In the future when most users have updated to newer versions that use the capability flag, we could change the code to use `enabledModelSelection`